

// const fetchData =()=>{
//     fetch(url)
//      .then((data)=>{
//     // after getting data success
//     })
//     .catch((err)=>{
//         // error handling
//     })
// }

// const test = async () => {

// }

// const fetchData = async ()=>{
//     try{
//         const user = await fetch(url)
//         const post = await fetch(getPost,user.id)
//         const sum = 10 + 9;
//         const sum2 = await test();
//     }catch(err){

//     }
// }




// const fetchData =()=>{
//     fetch(url)
//      .then((user)=>{
//          fetch('userPost',{user.id})
//          .then((post)=>{
//              console.log(posts)
//          })
//          .catch((err)=>{
//              console.log(err)
//          })
//     })
//     .catch((err)=>{
//         // error handling
//     })
// }
